doc/configuration.md

```diff
@@ -105,14 +105,27 @@ Disables fetching pool offchain metadata.
 
 Disables all data related to governance
 
-### --disable-all
+### --disable-all : Experimental
 
 Disables almost all data except `block`, `tx` and data related to the ledger state
 
+### --dont-use-ledger : Experimental
+
+Maintains the ledger state, but doesn't use any of its data, except to load UTxO. To be used with `--bootstrap-tx-out`
+
+### --only-utxo : Experimental
+
+This is the equivalent of using `--dont-use-ledger`
+`--disable-shelley`, `--disable-plutus-extra`, `--disable-offchain-pool-data`, `--disable-gov`, `--bootstrap-tx-out`.
+This mode initially populates only a few tables, like `block` and `tx`. It maintains a ledger state but doesn't use any of its data. When syncing is completed, it loads the whole UTxO set from the ledger to the `tx_out` and `ma_tx_out` tables.
+After that db-sync can be restarted with `--disable-ledger-state` to continue
```

Comment:
Can this be somehow simplified, it is a little hacky for operations/automations.
As against that, maybe the flag combinations could be adapted to extract UTxOs from ledger-state once upon booting initially, and then not rely on it beyond. This could potentially even be controlled against a control table (eg: config) within database. At the moment, the number of command line options and permutations seem to be getting a bit out-of-hand, especially from testing point of view

Reply:
> Can this be somehow simplified, it is a little hacky for operations/automations.

Indeed, that's possible, it requires a somewhat big refactoring, to make the LedgerEnv a mutable var, but it's something we will eventually do.

> At the moment, the number of command line options and permutations seem to be getting a bit out-of-hand, especially from testing point of view.

Testing/QA is indeed a challenge with so many different ways to run db-sync. The plan is to have `modes`, which are common combinations of these flags, based on user input. We can then focus our testing efforts on these modes. `--only-utxo` which is introduced in this pr is an example of such mode which meets your requirement (except the ledger state automatic dropping which is not done in this pr).